app/Http/Middleware/ReminderIsValid.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use App\Reminder;
+use Closure;
+use Illuminate\Support\Facades\Response;
+
+class ReminderIsValid
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $reminder = Reminder::findOrFail($request->reminder->id);
+
+        if ($reminder === null) {
```

Comment:
This is redundant, as the `findOrFail()` will throw an `Exception` and the code won't progress if `$reminder` is `null`.